src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,24 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
+    private int radius;
+    private String className = this.getClass().getSimpleName();
+
+    public Circle(int radius, String color) {
+        super(color);
+        this.radius = radius;
+    }
+
+    @Override
+    public double getArea() {
+        return Math.PI * radius * radius;
+    }
+
+    @Override
+    public String drawing() {
+        return "Figure: " + className + ", "
+                + "area: " + getArea() + " sq. units, "
+                + "radius: " + radius + " units, "
+                + "color: " + getColor();
+    }
```

Comment:
There is no need to extract to class variable rather simple expression that is used in a single method for a single time
```suggestion
public class Circle extends Figure {
    private int radius;

    public Circle(int radius, String color) {
        super(color);
        this.radius = radius;
    }

    @Override
    public double getArea() {
        return Math.PI * radius * radius;
    }

    @Override
    public String drawing() {
        return "Figure: " + getClass().getSimpleName() + ", "
                + "area: " + getArea() + " sq. units, "
                + "radius: " + radius + " units, "
                + "color: " + getColor();
    }
```
Please recheck other classes for same issue